tests/integration/test_epicor.py

```diff
@@ -0,0 +1,50 @@
+import pytest
+
+from viadot.sources import Epicor
+from viadot.config import local_config
+from viadot.exceptions import DataRangeError
+
+
+@pytest.fixture(scope="session")
+def epicor():
+    epicor = Epicor(
+        base_url=local_config.get("Epicor").get("test_url"),
+        config_key="Epicor",
```

Comment:
I would use all caps as for all other cred keys (except DuckDB, not sure what happened there :D)